src/Database/Esqueleto.hs

```diff
@@ -129,7 +130,7 @@ module Database.Esqueleto {-# WARNING "This module will switch over to the Exper
   ) where
 
 import Database.Esqueleto.Legacy
-import Database.Esqueleto.Internal.PersistentImport
+import Database.Esqueleto.Internal.PersistentImport hiding (selectFirst)
```

Comment:
Instead of hiding the import, we should just not re-export it from `PersistentImpot`